.gitignore

```diff
@@ -1 +1,2 @@
 .env
+.coverage*
```

Comment:
Hey, Jo.
I think it's best to also include the coverage.xml in your .gitignore. We don't want the result of failing tests on your local to be uploaded too.

Reply:
Hi, Alex. Thank you for feedback. I will resolve this in the next commit along with adding the other files that needs to be properly gitignored.